docs/docs/recipes/create-your-connector/README.md

```diff
@@ -3,3 +3,15 @@ sidebar_position: 4
 ---
 
 # 🧑‍🔬 Create your connector
+
+Connectors play a critical role in Logto. With their help, Logto enables end-users to use SMS-based and email-based passwordless registration and sign-in capabilities using social accounts. With the increasing popularity of websites and applications, passwordless and social logins allow users to avoid managing numerous accounts and passwords every day.
```

Comment:
sign-ins?
```suggestion
Connectors play a critical role in Logto. With their help, Logto enables end-users to use SMS-based and email-based passwordless registration and sign-in capabilities using social accounts. With the increasing popularity of websites and applications, passwordless and social sign-ins allow users to avoid managing numerous accounts and passwords every day.
```